migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs

```diff
@@ -59,6 +59,17 @@ impl SqlRenderer for PostgresFlavour {
                 })
                 .collect();
 
+            if stmts.len() > 1 {
+                let warning = indoc::indoc! {
+                    r#"-- This migration adds more than one variant into an enum.
```

Comment:
Is variants the word we use for this at Prisma? We seem to use values in the docs. See https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#enum

```suggestion
                    r#"-- This migration adds more than one value to an existing enum.
```